packages/back-end/src/scim/users/updateUser.ts

```diff
@@ -0,0 +1,11 @@
+import { Request } from "express";
+import { ApiRequestLocals } from "../../../types/api";
+
+export async function updateUser(
+  req: Request & ApiRequestLocals,
+  res: Response
+): Promise<Response> {
+  // TODO: Implement user attribute update
```

Comment:
I took a pass at this - I *think* we'll be safe to update the email and the name, even though that isn't really a concept we. have in GB due to the multi-tenancy. But, since this will be tied to SSO, it should be ok.